backend/cypress/pageObjects/general-info.js

```diff
@@ -0,0 +1,103 @@
+import auditeePhoneField from '../fixtures/genInfoData/auditeePhoneField.json';
+import auditeeEmailField from '../fixtures/genInfoData/auditeeEmailField.json';
+import auditorEINField from '../fixtures/genInfoData/auditorEINField.json';
+import auditorFirmNameField from '../fixtures/genInfoData/auditorFirmNameField.json';
+import auditorAddressLine1Field from '../fixtures/genInfoData/auditorAddressLine1Field.json';
+import auditorCityField from '../fixtures/genInfoData/auditorCityField.json';
+import auditorStateField from '../fixtures/genInfoData/auditorStateField.json';
+import auditorZipField from '../fixtures/genInfoData/auditorZipField.json';
+import auditorContactNameField from '../fixtures/genInfoData/auditorContactNameField.json';
+import auditorContactTitleField from '../fixtures/genInfoData/auditorContactTitleField.json';
+import auditorPhoneField from '../fixtures/genInfoData/auditorPhoneField.json';
+import auditorEmailField from '../fixtures/genInfoData/auditorEmailField.json';
+
+
+class GeneralInfoForm {
+    constructor() {
+        this.auditTypeSingle = 'label[for=single-audit]';
+        this.auditPeriodAnnual = 'label[for=audit-period-annual]';
+        this.auditeeNameField = '#auditee_name';
+        this.einField = '#ein';
+        this.einNotSSNAttestationLabel = 'label[for=ein_not_an_ssn_attestation]';
+        this.multipleEINsYesLabel = 'label[for=multiple-eins-yes]';
+        this.auditeeAddressLine1Field = '#auditee_address_line_1';
+        this.auditeeCityField = '#auditee_city';
+        this.auditeeStateField = '#auditee_state';
+        this.auditeeZipField = '#auditee_zip';
+        this.auditeeContactNameField = '#auditee_contact_name';
+        this.auditeeContactTitleField = '#auditee_contact_title';
+        this.auditeePhoneField = '#auditee_phone';
+        this.auditeeEmailField = '#auditee_email';
+        this.auditorEINField = '#auditor_ein';
+        this.auditorEINNotSSNAttestationLabel = 'label[for=auditor_ein_not_an_ssn_attestation]';
+        this.auditorFirmNameField = '#auditor_firm_name';
+        this.auditorAddressLine1Field = '#auditor_address_line_1';
+        this.auditorCityField = '#auditor_city';
+        this.auditorStateField = '#auditor_state';
+        this.auditorZipField = '#auditor_zip';
+        this.auditorContactNameField = '#auditor_contact_name';
+        this.auditorContactTitleField = '#auditor_contact_title';
+        this.auditorPhoneField = '#auditor_phone';
+        this.auditorEmailField = '#auditor_email';
+        this.continueButton = '#continue';
+        this.auditeeUEI = '#auditee_uei';
+        this.multipleUEIsYesLabel = 'label[for=multiple-ueis-yes]';
+        this.secondaryAuditorsField = 'label[for=secondary_auditors-yes]';
+    }
+    
+
+    selectAuditTypeAndPeriod() {
+        cy.get(this.auditTypeSingle).click();
+
+        cy.get(this.auditPeriodAnnual).click();
+    }
+
+    fillAuditeeInformation() {
```

Comment:
Wondering if there's a way to separate the page objects from the fixture data, such that the fixture data gets passed into these `fill` functions rather than loaded internally. Relatedly, is there a reason we can't have a single `json` file per page (e.g. `genInfo.json`, `search.json`, etc) rather than one file per field per page. That way, these functions could look more like:
```
fillAuditeeInformation(genInfo) {
  cy.get(this.auditeeNameField).type(genInfo.auditeeNameField);
  ...
}
```

thoughts?

Reply:
1. We can seperate the fixture data. I was following the search POM but it had its own search but gen-info is just being passed into other tests.
2. We can have a single json file. Again I was trying to make it consistent with Search POM fixtures.

I can start to implement those changes and request re-review.

Reply:
We now have one file now for genInfoData and also separated page objects from the fixture data.